js/projections.js

```diff
@@ -20,6 +20,7 @@
 		factory(Highcharts);
 	}
 }(function (H) {
+	H.perspective = H._modules['Core/Math3D.js'].perspective;
```

Comment:
This method used to be on the Highcharts object (H.) then it was moved to a new module. Now there's no point to store it on the Highcharts object again. Instead, a simple local variable would do the job, like:
```js
const perspective = ...;
```